ServerStatisticsPublisher/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using ServerStatistics;
+using MessageQueue;
+using GlobalConfigurations;
+
+var serverStatistics = new ServerStatisticsCollector();
+var config = new AppConfigurations();
+var samplingInterval = config.ServerStatisticsConfig.SamplingIntervalSeconds;
+var serverIdentifier = config.ServerStatisticsConfig.ServerIdentifier;
+
+Console.WriteLine(
+    "This service is responsible for sending the server statistics to a message queue.\n");
+
+Console.WriteLine("Connecting...");
+TopicMessageSender? messageSender = null;
+
+while (true) {
+
+    try
+    {
+        messageSender = new TopicMessageSender(
+            "Message sending",
+            "Statistics exchange",
```

Comment:
these constants such queue names would be better to be moved to a different class in this project, same thing for the other constants in the other projects